src/main/java/seedu/storemando/logic/commands/ReminderCommand.java

```diff
@@ -16,7 +15,9 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Filters all items whose expiry date is within "
         + "the user-specified number of days from the current date and displays them as a list with index numbers.\n"
         + "Parameters: numOfDays (must be a positive integer) [timeUnit] (must either be days or weeks)\n"
-        + "Example: \"" + COMMAND_WORD + " 3\" or \"" + COMMAND_WORD + " 2 weeks\"";
+        + "Example: " + COMMAND_WORD + " 3 days";
```

Comment:
Can we add one example for weeks as well? :)

Reply:
All right!